lib/internal/util/inspect.js

```diff
@@ -241,14 +241,17 @@ const meta = [
 ];
 
 // Regex used for ansi escape code splitting
-// Adopted from https://github.com/chalk/ansi-regex/blob/HEAD/index.js
-// License: MIT, authors: @sindresorhus, Qix-, arjunmehta and LitoMore
+// Ref: https://github.com/chalk/ansi-regex/blob/f338e1814144efb950276aac84135ff86b72dc8e/index.js
+// License: MIT by Sindre Sorhus <sindresorhus@gmail.com>
```

Comment:
**Review note**: I've changed this from an authors list, as the number of authors has grown since this was last updated.

Reply:
Yes

Reply:
Hey